scripts/upload-layouts.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/sh
+
+echo "Upload Shelf Layouts"
+
+directory=$1
```

Comment:
Should we have a check to see if it has been set and if not, abort early with an error message?